File: algorithms/palindrome_and_get_digits..js
function getDigits(str) {
    let newString = ''
    for(let i = 1; i <str.length; i++){
        if(str[i] >= 0 && str[i] <= 9){
            newString += str[i]
        }
    }
    return newString
}
console.log(getDigits('z4d2g5s2'))

// should log 4252

function isPalindrome(str) {
    let newString = ''
    for( let i = str.length - 1; i >= 0; i--){
        newString += str[i]
    }
    if(str === newString){
        return true
    }
        return false
}
console.log(isPalindrome('Tuesday'))
// false

console.log(isPalindrome('tacocat'));
// true

File: algorithms/palindrome_and_get_digits.py
#
# 
#

# def getDigits(str):
#     newStr = []
#     for i in range(0, len(str)):
#         if type(str[i]) == int:
#             if str[i] % 2 == 0 or str[i] % 2 == 1:
#                 newStr.append(str[i])
#     return newStr

# print(getDigits("[z4d2g5s2]))

# def gotDigits(str):
#     newStr = ""
#     for i in range(len(str)):
#         if str[i] >= "0" and str[i] <= "9":
#                 newStr += str[i]
#     return newStr

# print(gotDigits("z4d2g5s2"))

def isPalindrome(str):
    isPal = True
    for i in range(len(str)):
        if str[i] == str[len(str) - 1 - i]:
            isPal = True
        else: 
            isPal = False
    return isPal

print(isPalindrome('tuesday'))
print(isPalindrome('tacocat'))

